phpBB/adm/style/tooltip.js

```diff
@@ -10,206 +10,173 @@ phpBB Development Team:
 	- further adjustements
 */
 
-var head_text, tooltip_mode;
+(function($) { // Avoid conflicts with other libraries
 
-/**
-* Enable tooltip replacements for links
-*/
-function enable_tooltips_link(id, headline, sub_id) {
-	var links, i, hold;
-
-	head_text = headline;
-
-	if (!document.getElementById || !document.getElementsByTagName) {
-		return;
-	}
-
-	hold = document.createElement('span');
-	hold.id = '_tooltip_container';
-	hold.setAttribute('id', '_tooltip_container');
-	hold.style.position = 'absolute';
+'use strict';
 
-	document.getElementsByTagName('body')[0].appendChild(hold);
-
-	if (id === null) {
-		links = document.getElementsByTagName('a');
-	} else {
-		links = document.getElementById(id).getElementsByTagName('a');
-	}
-
-	for (i = 0; i < links.length; i++) {
-		if (sub_id) {
-			if (links[i].id.substr(0, sub_id.length) === sub_id) {
-				prepare(links[i]);
-			}
-		} else {
-			prepare(links[i]);
-		}
-	}
-
-	tooltip_mode = 'link';
-}
+var tooltips = [];
 
 /**
-* Enable tooltip replacements for selects
+ * Enable tooltip replacements for selects
+ * @param {string} id ID tag of select
+ * @param {string} headline Text that should appear on top of tooltip
+ * @param {string} subId Sub ID that should only be using tooltips (optional)
```

Comment:
For optional params, wrap the name in square brackets:

`@param {string} [subId] Sub ID that should only be using tooltips`

http://usejsdoc.org/tags-param.html#optional-parameters-and-default-values